### PR TITLE
add comments to modules in auction demo keeper

### PR DESCRIPTION
**Pull request by Khay-EMMA**

### CODE REVIEW

File: src/clipper.js [hunk 1]
```diff
@@ -30,12 +31,18 @@ export default class Clipper {
     //TODO: start timer every minute
   }
 
+  // Initialize the clipper
   async init() {
 
+    // initialize the clipper contract object
     this._clipper = new ethers.Contract(this._clipperAddr, clipperAbi, network.provider);
+
     this._abacusAddr = await this._clipper.calc();
```
Comment (pcatana) [inline src/clipper.js:40]: add comment:
// _clipper.calc() returns the abacus address of the collateral

File: src/keeper.js [hunk 2]
```diff
@@ -99,6 +107,7 @@ export default class keeper {
       if(Object.prototype.hasOwnProperty.call(Config.vars.collateral,name)) {
         const collateral = Config.vars.collateral[name];
         this._clipperInit(collateral).then(pair => {
+          // add the pair to the array of clippers
```
Comment (pcatana) [inline src/keeper.js:110]: Can we find out what is the pair variable actually defined to? I'm personally curious :))

File: src/clipper.js (hunk 1 above)
Comment (pcatana) [inline src/clipper.js:40]: add comment:
// _clipper.calc() returns the abacus address of the collateral

**Review by pcatana — CHANGES_REQUESTED** [review]
Requested an extra comment on 2 lines of code.

File: src/keeper.js (hunk 2 above)
Comment (Khay-EMMA) [reply src/keeper.js:110]: Alright!

**Review by pcatana — APPROVED** [review]
LGTM

**Comment by Khay-EMMA** [comment]
> LGTM

what's LGTM?

**Pull request merged**